Responsive design implementation

mistermaxis-zz opened the pull request:
I created the project in the main branch. That was a mistake. Also, the project is not yet finished.

mistermaxis-zz committed:
Fix indentation issues with linter

mistermaxis-zz committed:
Fix issues with webhint

mke2111 reviewed (changes requested):
# STATUS: INVALID

I'm invalidating this PR because of the following reason:

- [ ] Kindly add a brief description to this PR.

### Kindly make all the requested changes and submit another CRR. 💯 

**Cheers 🍹 and Happy coding 💻** 

## Reach me on slack for a chit-chat 🆘  Roy Mukuye

mistermaxis-zz committed:
Fix horizontal scrollbar

Set elements inheriting visibility: hidden property to width and height zero so they don't overflow and create horizontal scrollbar.
